src/vs/base/browser/htmlContentRenderer.ts

```diff
@@ -143,7 +144,7 @@ export function renderMarkdown(markdown: IMarkdownString, options: RenderOptions
 	}
 
 	if (options.actionHandler) {
-		options.actionHandler.disposeables.push(DOM.addStandardDisposableListener(element, 'click', event => {
+		options.actionHandler.disposeables.push(DOM.addStandardDisposableListener(element, options.eventToIntercept || 'click', event => {
```

Comment:
This change is sloppy.

Reply:
If this was just to support links then I think we can remove it.